crates/l2/Makefile

```diff
@@ -9,7 +9,10 @@ L1_GENESIS_FILE_PATH=../../test_data/genesis-l1-dev.json
 help: ## 📚 Show help for each of the Makefile recipes
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-init: init-local-l1 deploy-l1 update-system-contracts  init-l2 ## 🚀 Initializes a localnet with Lambda ethrex client as both L1 and L2
+init: init-local-l1 deploy-l1 update-system-contracts init-l2 ## 🚀 Initializes a localnet with Lambda ethrex client as both L1 and L2
+
+## Same as init but does not do deposits for rich accounts since that doesn't make sense for deployments to devnets
+init-devnet: init-local-l1 deploy-l1-devnet update-system-contracts  init-l2
```

Comment:
Rename this to `testnet` since we'd want to avoid deploying with with rich wallet deposits when we're deploying on testnets.